spec/spec_helper.rb

```diff
@@ -6,4 +6,12 @@
   config.expect_with :rspec do |expectations|
     expectations.include_chain_clauses_in_custom_matcher_descriptions = true
   end
+
+  config.backtrace_exclusion_patterns = [
+    /\/lib\d*\/ruby\//,
+    /bin\//,
+    /gems/,
+    /spec\/spec_helper\.rb/,
+    /lib\/rspec\/(core|expectations|matchers|mocks)/
+  ]
```

Comment:
Ah Cool, thats super handy. Im going to steal that for a lot of projects thanks!